### DISCUSSION TITLE
How to listen when user react to message?

### DISCUSSION THREAD

**Discussion by mikenlanggio** [yagop/node-telegram-bot-api#1186] [Q&A] [answered]
From telegram api 7.0, MessageReactionUpdated added.
How can I use from this library?

**Comment by danielperez9430** [comment, marked as answer]
You need to include the `message_reaction` in the list of the `allowed_updates` list

This is a example of how to listing this event;

```js
bot.on('message_reaction', (msg) => {
    const chatId = msg.chat.id;
    console.log(msg);
    bot.sendMessage(chatId, 'Received your reaction message');
});

```
